This adds a missing Component.h #include to ComponentOutput so that downstream
code can include it without any issues.

Specifically, downstream code cannot compile with MSVC if it only includes
the ComponentOutput.h header because OpenSim::Output<T>::Channel::getPathName
calls functions from the Component API. Other compilers might let this through
because Output<T> is a template that may not actually be instantiated.
---
diff --git a/OpenSim/Common/ComponentOutput.h b/OpenSim/Common/ComponentOutput.h
index 62ea6cf53..ec6f35891 100644
--- a/OpenSim/Common/ComponentOutput.h
+++ b/OpenSim/Common/ComponentOutput.h
@@ -31,6 +31,7 @@
  */
 
 // INCLUDES
+#include "Component.h"
 #include "Exception.h"
 #include "Object.h"
 
@@ -42,7 +43,6 @@
 
 namespace OpenSim {
 
-class Component;
 class AbstractInput;
 
 /** One of the values of an Output. */
